lib/cadet/assessments/assessments.ex

```diff
@@ -488,16 +488,17 @@ defmodule Cadet.Assessments do
       |> where(submission_id: ^id)
       |> join(:inner, [a], q in assoc(a, :question))
       |> join(:left, [a, _], g in assoc(a, :grader))
-      |> preload([_, q, g], question: q, grader: g)
+      |> join(:inner, [a, _, _], s in Submission, a.submission_id == s.id)
+      |> join(:inner, [a, _, _, s], st in User, s.student_id == st.id)
```

Comment:
Prefer `|> join(:inner, [a, ..., s], st in assoc(s, :student))`

Reply:
Erm I think you missed this one

Reply:
opps my bad! made the changes.